.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
What's this file for ?

Reply:
It can bé used to join a container from local vscode. It's generated by the latest versions kubebuilder so I added it

Reply:
Oh ok